src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import absoulute_import
+import sys
+
+from google.gax.utils.messages import get_messages
+
+from google.api import http_pb2
+from google.cloud.proto.example.library.v1 import book_from_anywhere_pb2
+from google.cloud.proto.example.library.v1 import field_mask_pb2 as v1_field_mask_pb2
+from google.cloud.proto.example.library.v1 import library_pb2
+from google.cloud.proto.tagger.v1 import tagger_pb2
+from google.cloud.proto.test.shared.data import other_shared_type_pb2
+from google.cloud.proto.test.shared.data import shared_type_pb2
+from google.longrunning import operations_pb2
+from google.protobuf import any_pb2
+from google.protobuf import descriptor_pb2
+from google.protobuf import duration_pb2
+from google.protobuf import empty_pb2
+from google.protobuf import field_mask_pb2 as protobuf_field_mask_pb2
+from google.protobuf import struct_pb2
+from google.protobuf import timestamp_pb2
+from google.protobuf import wrappers_pb2
+from google.rpc import status_pb2
+
+
+names = []
+for module in (http_pb2
+               book_from_anywhere_pb2
+               v1_field_mask_pb2
+               library_pb2
+               tagger_pb2
+               other_shared_type_pb2
+               shared_type_pb2
+               operations_pb2
+               any_pb2
+               descriptor_pb2
+               duration_pb2
+               empty_pb2
+               protobuf_field_mask_pb2
+               struct_pb2
+               timestamp_pb2
+               wrappers_pb2
+               status_pb2):
+    for name, message in get_messages(module).items():
+        message.__module__ = 'google.cloud.proto.example.library_v1'
```

Comment:
This line is _probably_ wrong? I would expect it to be `google.cloud.library_v1`.